### PR TITLE
feat: add example to iotda server

### PR DESCRIPTION
**Pull request by zhiheng123**

### CODE REVIEW

File: mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
+
+        MTConnectClientConfiguration configuration = new MTConnectClientConfiguration();
+        configuration.setHost("127.0.0.1");
+        configuration.setPort(36633);
+        HttpClientConfig httpClientConfig = new HttpClientConfig.Builder().build();
+        configuration.setHttpConfig(httpClientConfig);
+        MTConnectClient mtConnectClient = new MTConnectClient(configuration);
+
+        MTConnectDevices devices = mtConnectClient.devices();
+        System.out.println(XmlUtil.toXml(devices));
```
Comment (hezhangjian) [inline mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java]: use `log.info` instead

Comment (zhiheng123) [reply mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java]: At first, I thought that `example` should use `printIn` to print to the console for more intuitive use, while log4j is more for running logs and other outputs. Now I am a little hesitant. We can discuss this issue in detail.

File: mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import io.github.protocol.mtconnect.server.MTConnectServerConfiguration;
 import io.github.protocol.mtconnect.server.impl.IoTDAMtProcessor;
 
+
```
Comment (hezhangjian) [inline mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectIoTDAServerExample.java]: unnecessary blank line

File: mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectClientExample.java [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
         MTConnectClientConfiguration configuration = new MTConnectClientConfiguration();
+        configuration.setHost("127.0.0.1");
+        configuration.setPort(36633);
+
         HttpClientConfig httpClientConfig = new HttpClientConfig.Builder().build();
         configuration.setHttpConfig(httpClientConfig);
         MTConnectClient mtConnectClient = new MTConnectClient(configuration);
+
+        MTConnectDevices devices = mtConnectClient.devices();
+        System.out.println(XmlUtil.toXml(devices));
```
Comment (hezhangjian) [inline mtconnect-examples/src/main/java/io/github/protocol/mtconnect/examples/MTConnectClientExample.java]: use log.info instead

**Pull request closed without merging**